src/app/homepage/homepage.component.scss

```diff
@@ -0,0 +1,85 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@mixin theme($theme) {
+  .recent-container {
+    height: 40vh;
+    padding-left:32px;
+    padding-right: 32px;
+    font: mat.get-theme-typography($theme, headline-large);
+  }
+
+  .user-sets-container{
+    height: 46vh;
+    padding-left:32px;
+    padding-right: 32px;
+  }
+  .recent-title{
+    font: mat.get-theme-typography($theme, headline-medium);
+    font-weight: bold;
+    line-height: 0em;
+  }
+  .recent-carousel{
+    overflow-x: scroll;
+  }
+  .recent-carousel-items{
+    margin-top: 4px;
+    list-style-type: none;
+    height:252px;
+    width:max-content;
+  }
```

Comment:
The shadow on the first study set card get cut off when the page first loads. 
<img width="1377" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/99c679b1-2423-43c7-82b2-41eec80e054a">

You can add padding to the left margin to prevent this or come up with a more clever technique.